BraintreeDropIn/Public/BTDropInRequest.h

```diff
@@ -58,6 +58,14 @@ typedef NS_ENUM(NSInteger, BTFormFieldSetting) {
 /// Defaults to false.
 @property (nonatomic, assign) BOOL vaultManager;
 
+/// Optional: Whether or not to vault the card upon tokenization. When set to `false` with `allowVaultCardOverride` set to `false`, then cards will not be vaulted.
+/// Defaults to true
+@property (nonatomic, assign) BOOL vaultCard;
+
+/// Optional: When true, the card form will include an option to let the customer decide not to vault the card.
```

Comment:
Do we want to be explicit that this setting only works when using a client token with a customer?

Reply:
Good call out! Clarified the description accordingly.